projects/m1/026-is-a-string-a-palindrome/solutions/alessiacaporaso/is-a-string-a-palindrome.js

```diff
@@ -0,0 +1,18 @@
+// enter string
+const enterString = prompt('Please enter the string to check');
+// divide string in the array
+const divideString = enterString.split("");
+console.log(divideString)
+// reverse array
+const reverseString = divideString.reverse()
+console.log(reverseString)
+// join array
+const joinString = reverseString.join("")
+console.log(joinString)
+// check the string and if enter string is equal to the reverse string return palindrome
+if(enterString == joinString){
```

Comment:
Be careful here.
https://www.freecodecamp.org/news/javascript-triple-equals-sign-vs-double-equals-sign-comparison-operators-explained-with-examples/